pyproject.toml

```diff
@@ -17,6 +17,7 @@ httpx = "^0.16.1"
 gunicorn = "^20.0.4"
 pydantic = "^1.7.2"
 pydnsbl = "^1.1"
+pytest = "^6.2.0"
```

Comment:
Should we be adding pytest in this PR? Looks like the changes to poetry.lock and pyproject.toml are related but really we should be adding this in a PR which introduces not only this but CI tests as well.